otherother 2.txt

```diff
@@ -0,0 +1 @@
+This file is more important than my life
```

Comment:
Delete all other files not related to this PR.

Reply:
Deleted all files except the folder containing my codingame solution.